src/themes/ec/mixins/_typography.scss

```diff
@@ -0,0 +1,70 @@
+@use 'sass:map';
+@use 'sass:string';
+@use '@ecl/grid/mixins/breakpoints';
+
+$font-size-map: (
```

Comment:
if I'm not wrong, there is a link between the font size and the font weight, for the designers.
If that is the case, we could have a default value for the font weight based on another mapping ('3xl': 300, for instance). 
We would still be able to override the value in the mixin, but for standard case we would not have to define it each time